src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -566,7 +640,7 @@ def translate_symbol_ref(
 if TYPE_CHECKING:
```

Comment:
Why is that thing at the end of the file?
According to the coding convention it should be at the top.

Reply:
According to the [google style guide](https://google.github.io/styleguide/pyguide.html#31913-conditional-imports) such imports should be at the top.
But I am not sure if this is really needed at all.

Reply:
But a conditional declaration does not make sense to me.

Reply:
The `TYPE_CHECKING` flag is only `True` while we run mypy. In regular python execution, `TYPE_CHECKING` is `False` and the variable `__primitive_translators` is not declared.